examples/drivers/windows/otApi/otApi.cpp

```diff
@@ -2168,7 +2168,7 @@ OTCALL
 otSendActiveSet(
     _In_ otInstance *aInstance, 
     const otOperationalDataset *aDataset, 
-    const uint8_t *aTlvs,
+    uint8_t *aTlvs,
```

Comment:
`otSendActiveSet()` calls `SendSetRequest()`, define a pointer `cur` to point the beginning of TLV buffer `aTlvs` which is a const variable, cannot convert const `unsigned char *` to `uint8_t *` type, however, we need to modify the `cur` pointer to find the session id tlv, should not define `cur` with const modifier neither, so remove the `const` modifier from `SendSetRequest()` and upper level API that call this function.